README.md

```diff
@@ -0,0 +1,70 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math Magician
+
+> Description the project is incomming.
+
+
+## Built With
+
+- React, Webpack, Javascript, and HTML/CSS
+- NULL
+- NODEJS, LINTERS.
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com) incomming
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- Optional.
Kindly update these sections as well, and the sections which you feel are not relevant to this project, you can remove them to keep the README file clean.